src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Error.cshtml

```diff
@@ -14,8 +16,7 @@
     <div class="container">
         <div class="row">
             <div class="col text-center">
-                <h1 class="mb-30">Ups... Coś poszło nie tak...</h1>
-                <p>Nasze elfy już nad tym pracują ;-)</p>
+                @Model.Message
```

Comment:
could you use h1 tag here?

Reply:
and update tests, too :) You can use String.contains(), not compare whole html text

Reply:
You mean move h1 tag from resources here?

Reply:
no - put the @model.Message value in h1 tag

Reply:
I added h1 tag, and used contains in tests